docs/general/todo.md

```diff
@@ -1,10 +1,13 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
-# The SeAT TODO list!
+# Todo
 
-This list is supposed to give a very brief idea of *whats coming™*. It is the stuff that should be developed; should be improved; should be fixed etc. All items in this list is very high level and should be interpreted from a long term perspective. Of course, there are also the Github tickets tagged as _wishlist_ items: [https://github.com/eveseat/seat/labels/wishlist](https://github.com/eveseat/seat/labels/wishlist).
+This list is supposed to give a very brief idea of *whats coming™*.
+It is the stuff that should be developed; should be improved; should be fixed etc.
+All items in this list is very high level and should be interpreted from a long term perspective.
+Of course, there are also the Github tickets tagged as _wishlist_ items: [https://github.com/eveseat/seat/labels/wishlist](https://github.com/eveseat/seat/labels/wishlist).
 
```

Comment:
i would also encourage user to submit their wished items 👍  that could be fun see what community would like to see